nautobot_design_builder/models.py

```diff
@@ -150,8 +151,6 @@ def get_by_natural_key(self, design_name, instance_name):
 
 DESIGN_NAME_MAX_LENGTH = 100
```

Comment:
I know that this isn't a change in the current branch/PR but Nautobot core has standardized on a length of 255 for all char fields. Maybe we should as well.